.github/workflows/update-test.yaml

```diff
@@ -8,12 +8,23 @@ on:
       - prerelease_test
   pull_request:
 jobs:
+  config:
+    runs-on: ubuntu-latest
+    outputs:
+      pg_latest: ${{ steps.setter.outputs.PG_LATEST }}
+    steps:
+    - name: Checkout source code
+      uses: actions/checkout@v2
+    - name: Build matrix
+      id: setter
+      run: python .github/gh_config_reader.py
+
   update_test:
     name: Update test PG${{ matrix.pg }}
     runs-on: 'ubuntu-latest'
```

Comment:
Most other GitHub actions that are using `gh_config_reader.py` have a dependency on the new `config` job (`needs: config`). From my point of view, this would also make sense for the `update_test` job.